lampod/src/ln/inventory_manager.rs

```diff
@@ -33,11 +33,32 @@ impl InventoryHandler for LampoInventoryManager {
         match event {
             InventoryCommand::GetNodeInfo(chan) => {
                 let chain = self.channel_manager.conf.network.to_string();
+                let alias = self
+                    .channel_manager
+                    .conf
+                    .alias
+                    .as_ref()
+                    .map(|alias| alias.to_string());
+                // we have to put "" in case of alias missing as cln provide us with a random alias.
+                let alias = alias.unwrap_or_else(|| "".to_string());
+                let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
+                let blockheight;
+                if height.is_none() {
+                    blockheight = 0
+                } else {
+                    // Safe to unwrap here
+                    blockheight = height.unwrap();
+                }
+                let lampo_dir = self.channel_manager.conf.root_path.to_string();
                 let getinfo = GetInfo {
                     node_id: self.channel_manager.manager().get_our_node_id().to_string(),
                     peers: self.peer_manager.manager().list_peers().len(),
                     channels: self.channel_manager.manager().list_channels().len(),
                     chain,
+                    alias,
+                    blockheight,
+                    warnings: self.channel_manager.onchain.backend.get_warnings().unwrap(),
```

Comment:
```suggestion
```

nit: this is excessive, if there is no good way to add warning, we do not add it. Do not overcomplicate the code

Reply:
Yes, I also realised that this is wrong. The warnings are from the bitcoin node and should not be put up here. The main aim of this should be like the core-lightning one which only tells the warning in case of syncing the bitcoin.